lib/v2/screens/explore_screens/vote/interactor/viewmodels/vote_state.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/datasource/remote/model/proposals_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+/// --- STATE
+class VoteState extends Equatable {
+  final PageState pageState;
+  final Map<String, Map> proposalTypes;
+  final Map<String, String>? currentType;
+  final List<ProposalModel>? proposals;
+  final String? errorMessage;
+
+  const VoteState({
+    required this.pageState,
+    required this.proposalTypes,
+    this.currentType,
+    this.proposals,
+    this.errorMessage,
+  });
+
+  @override
+  List<Object?> get props => [
+        pageState,
+        currentType,
+        proposals,
+        errorMessage,
+      ];
+
+  VoteState copyWith({
+    PageState? pageState,
+    List<ProposalModel>? proposals,
+    Map<String, String>? currentType,
+    String? errorMessage,
+  }) {
+    return VoteState(
+      pageState: pageState ?? this.pageState,
+      proposalTypes: proposalTypes,
+      currentType: currentType ?? this.currentType,
+      proposals: proposals,
+      errorMessage: errorMessage ?? this.errorMessage,
+    );
+  }
+
+  factory VoteState.initial() {
+    return const VoteState(
+      pageState: PageState.initial,
+      proposalTypes: {
+        'Open': {'stage': 'active', 'status': 'open'},
+        'Evaluate': {'stage': 'active', 'status': 'evaluate', 'reverse': 'true'},
+        'Passed': {'stage': 'done', 'status': 'passed', 'reverse': 'true'},
+        'Failed': {'stage': 'done', 'status': 'rejected', 'reverse': 'true'},
+      },
```

Comment:
I'm not really sure if to keep this in the state or if I want to keep it as a Map instead of a list of objects. 

I also need to ask (to Nick) what changes it will have since theses tab names do not match the v2 designs.

Reply:
yeeah, lets talk about this. I dont like the strings

Reply:
changed to a object list with the new filters